code/src/OSFramework/Feature/ICellStyle.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.Feature {
+    export interface ICellStyle {
+        addClass(binding: string, rowNumber: number, className: string): void;
```

Comment:
Please add some comments here in order to make it easier to understand the behavior.
Also, please check if we should declare the following methods as well: (not sure)
getMetadata
hasMetadata

Reply:
done